frontend/src/hooks/useSearch.tsx

```
@@ -19,15 +25,15 @@ import { useUrlQueryParam } from "./useUrlQueryParam";
 const buildOrParams = <T,>({ params, searchText }: TBuildParamProps<T>) => ({
   or: params?.map((field) => ({
     [field]: { contains: searchText },
-  })),
+  })) as OrParam<T>[],
 });
 const buildILikeParams = <T,>({ params, searchText }: TBuildParamProps<T>) =>
   params?.reduce(
     (acc, field) => ({
       ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)